src/test/java/org/folio/innreach/domain/service/impl/ContributionValidationServiceImplTest.java

```diff
@@ -303,16 +313,38 @@ void testEligibleInstance_statisticalCodeAllowed() {
     var instance = new Instance();
     instance.setStatisticalCodeIds(statisticalCodes);
     instance.setSource(ELIGIBLE_SOURCE);
-    instance.setItems(List.of(new Item().statisticalCodeIds(statisticalCodes)));
+    instance.setItems(List.of(new Item().statisticalCodeIds(statisticalCodes).effectiveLocationId(TEST_UUID)));
 
     when(contributionConfigService.getCriteria(any())).thenReturn(CRITERIA);
     when(holdingsService.find(any())).thenReturn(Optional.empty());
+    when(folioLocationService.getLocationLibraryMappings()).thenReturn(Map.of(TEST_UUID, TEST_UUID));
+    when(centralServerService.getCentralServer(any()))
+      .thenReturn(new CentralServerDTO().id(UUID.randomUUID()).localAgencies(List.of(
+        new LocalAgencyDTO().id(UUID.randomUUID()).folioLibraryIds(List.of(TEST_UUID)))));
```

Comment:
try to use CentralServerFixture.createCentralServer() here and for other tests

Reply:
added